### ISSUE TITLE
How is the two streams fused ?

### ISSUE THREAD

**Issue by CoinCheung** [bryanyzhu/two-stream-pytorch#47]
Hello, 

I noticed in the paper (https://arxiv.org/abs/1406.2199) that they finally used svm to fuse the two softmax scores computed by the two streams. Do you know how to use svm in this scenario?

**Comment by bryanyzhu** [comment]
Just think softmax scores as features, so each sample is just one row for SVM training. You can refer to this codebase for how to use SVM. https://github.com/YihengZhang-CV/SeCo-Sequence-Contrastive-Learning
